test/merge-patient-lab-iips-mocha.js

```diff
@@ -8,181 +9,146 @@ chai.use(chaiAsPromised);
 var _ = require('underscore');
 var noflo = require('noflo');
 
-var componentFactory = require('../components/merge-patient-lab-iips');
-var getComponent = componentFactory.getComponent;
+var compFactory = require('../components/merge-patient-lab-iips');
 
-var commonTest = require('./common-test');
+var test = require('./common-test');
 var commonStubs = require('./common-stubs');
```

Comment:
If you rename the commonTest to test, perhaps commonStubs should be simply stubs.